packages/admin/src/Charcoal/Admin/Template/System/ClearCacheTemplate.php

```diff
@@ -233,6 +251,57 @@ private function objectsCacheInfo()
         }
     }
 
+    private function twigCacheInfo()
+    {
+        $defaultCachePath = realpath($this->twigEngine->cache());
+        $cachePath = $defaultCachePath ? $defaultCachePath : realpath($this->appConfig['publicPath'] . DIRECTORY_SEPARATOR . $this->twigEngine->cache());
+        if (!is_dir($cachePath)) {
+            return [
+                'no_cache_folder' => true,
+            ];
+        }
+        $filesCount = new FilesystemIterator($cachePath, FilesystemIterator::SKIP_DOTS);
+        $folderSize = $this->dirSize($cachePath);
+
+        return [
+            'num_entries'     => iterator_count($filesCount),
+            'total_size'      => $this->formatBytes($folderSize),
+            'no_cache_folder' => false,
+        ];
+    }
+
+    private function mustacheCacheInfo()
+    {
+        $defaultCachePath = realpath($this->mustacheEngine->cache());
+        $cachePath = $defaultCachePath ? $defaultCachePath : realpath($this->appConfig['publicPath'] . DIRECTORY_SEPARATOR . $this->mustacheEngine->cache());
+        if (!is_dir($cachePath)) {
+            return [
+                'no_cache_folder' => true,
+            ];
+        }
+        $filesCount = new FilesystemIterator($cachePath, FilesystemIterator::SKIP_DOTS);
+        $folderSize = $this->dirSize($cachePath);
+
+        return [
+            'num_entries'     => iterator_count($filesCount),
+            'total_size'      => $this->formatBytes($folderSize),
+            'no_cache_folder' => false,
+        ];
+    }
+
+    /**
+     * Get the directory size
+     * @param  string $directory
+     * @return integer
+     */
+    private function dirSize($directory) {
+        $size = 0;
+        foreach (new RecursiveIteratorIterator(new RecursiveDirectoryIterator($directory)) as $file) {
+            $size += $file->getSize();
```

Comment:
The [`SplFileInfo::getSize()`](https://www.php.net/manual/en/splfileinfo.getsize.php) may return `false` on failure and may throw a `RuntimeException` if the file does not exist or an error occurs.

We should either cast the value to `int` to force `false` to `0` and maybe catch the exception to add `0` or skip the iteration.